testing/test_remote.py

```diff
@@ -118,7 +118,7 @@ def test_reprentries_serialization_170(self, testdir):
                             def test_repr_entry():
                                 x = 0
                                 assert x
-                        """, '--showlocals', '-n1')
+                        """, '--showlocals')
```

Comment:
As I commented before:

> Worked around them by not passing -n1 to them on the serialization-unittests given that we are only interested in obtaining the items and calling the serialization functions directly.

As to the actual underlying reason I have no idea. Seems like some caching is in place, because the second test session does not find any files in the `testdir` directory, even though the files are there.